system-tests/tests/smoke/cre/por_test.go

```diff
@@ -161,8 +161,7 @@ type DependenciesConfig struct {
 }
 
 const (
-	CronBinaryVersion   = "v1.0.2-alpha"
-	CRECLIBinaryVersion = "v0.1.5"
+	CRECLIProfile = "test"
```

Comment:
can we define it somewhere, where `system-tests/tests/smoke/cre/cmd/environment/environment.go:161` could also read it? or maybe pass it there?